smart_tests/utils/session.py

```diff
@@ -23,18 +23,58 @@ class TestSession:
     name: str | None = None
 
 
-def get_session(session: str, client: SmartTestsClient) -> TestSession:
-    build_name, test_session_id = parse_session(session)
-
-    subpath = f"builds/{build_name}/test_sessions/{test_session_id}"
-    res = client.request("get", subpath)
+class SessionId:
+    '''Represents the user-specific test session via the --session option.'''
+
+    def __init__(self, id: str):
+        '''This is the method in which we parse the user input, so be defensive'''
+        if id.startswith('@'):
```

Comment:
Could you add a test case for when the session option is prefixed with `@` (= file  path case).

Reply:
Added

Reply:
Thanks!